src/site/constants/drupals.js

```diff
@@ -2,7 +2,7 @@ const ENVIRONMENTS = require('./environments');
 
 // eslint-disable-next-line no-unused-vars
 const DRUPAL_DEV = {
-  address: 'http://dev.va.agile6.com',
+  address: 'http://dev.cms.va.gov',
```

Comment:
This and prod aren't actively used yet, so this shouldn't cause any problems to do early.